kolibri/plugins/style_guide/assets/src/views/content/breadcrumbs/index.vue

```diff
@@ -1,29 +1,30 @@
 <template>
 
-  <page-template title="Breadcrumbs" :completed="false">
+  <page-template :completed="true">
 
-    <h2><code>{{ kBreadcrumbsApi.name }}</code> API</h2>
-    <component-docs :api="kBreadcrumbsApi" />
+    <h1>Channel breadcrumbs</h1>
 
-    <h2>Code Example</h2>
-    <vue-example :code="example" />
+    <h2>Usage guidelines</h2>
 
-    <h2>Guidelines</h2>
+    <p>
+      Breadcrumbs hould be used to aid navigation with any instance of the topic tree.
+      Place it directly above the content to be navigated through.
+    </p>
 
-    <h3>Usage</h3>
-    <ul>
-      <li>Should be used with any instance of the topic tree.</li>
-      <li>Should be directly above the content intended to navigate through.</li>
-      <li>Should include the current item/location, which does not require a link.</li>
-    </ul>
+    <p>
+      It should include the current item/location, which does not require a link. (This
```

Comment:
The last item in the breadcrumbs should be the current item/location, which does not require a link.

Remove `This might be redundant with a header`, to enforce the rule above, even if redundant.

Reply:
👍

Reply:
sorry - don't quite understand the feedback. You're saying I should remove the phrase "This might be redundant with a header"?

Basically, I just wanted to clarify that it's OK for the text to be repeated, e.g. like

> ![image](https://user-images.githubusercontent.com/2367265/31413368-abcf6ac4-adcd-11e7-9a1f-f02d2b46574c.png)

Reply:
Just suggesting a slight rewording for clarity. `The last item in the breadcrumbs should always be the current item/location, even though it might seem redundant in certain cases.`  :smiley:

Reply:
Just saw your committed change :+1: